### PR TITLE
Fixes #9609: Deleted node should be periodically fully erased in LDAP (after some ttl)

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/9609

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala [hunk 1]
```diff
@@ -358,6 +379,7 @@ object RudderConfig extends Loggable {
   //val updateDynamicGroupsService : DynGroupUpdaterService = dynGroupUpdaterService
   val updateDynamicGroups: UpdateDynamicGroups = dyngroupUpdaterBatch
   val checkInventoryUpdate = new CheckInventoryUpdate(nodeInfoServiceImpl, asyncDeploymentAgent, stringUuidGenerator, 15.seconds)
+  val purgeDeletedInventories = new PurgeDeletedInventories(removeNodeServiceImpl, RUDDER_BATCH_PURGE_DELETED_INVENTORIES_INTERVAL.minute, RUDDER_BATCH_PURGE_DELETED_INVENTORIES)
```
Comment (fanf) [inline webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala]: shouldn't it be `hours` in place of  `minutes` here ?

Comment (ncharles) [reply webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala]: totally

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**